src/main/java/se/sundsvall/emailreader/integration/ews/EWSMapper.java

```diff
@@ -65,7 +85,33 @@ private Email.Attachment toAttachment(final FileAttachment fileAttachment) {
 			return null;
 
 		}
+	}
+
+	private Map<Header, List<String>> toHeaders(final EmailMessage emailMessage) {
+
+		try {
+			final var messageId = extractValues(emailMessage.getInternetMessageHeaders().find(MESSAGE_ID.getName()).getValue());
+			final var references = extractValues(emailMessage.getInternetMessageHeaders().find(REFERENCES.getName()).getValue());
+			final var inReplyTo = extractValues(emailMessage.getInternetMessageHeaders().find(IN_REPLY_TO.getName()).getValue());
+
+			return Map.of(
+				MESSAGE_ID, messageId,
+				REFERENCES, references,
+				IN_REPLY_TO, inReplyTo
+			);
+		} catch (final Exception e) {
+			log.warn("Could not load headers", e);
+			return Collections.emptyMap();
+		}
+	}
 
+	private List<String> extractValues(final String input) {
+		return Optional.ofNullable(input)
+			.map(inputString -> Pattern.compile(" ")
+				.splitAsStream(inputString)
+				.map(string -> string.replaceAll("[<>]", ""))
+				.toList())
+			.orElse(Collections.emptyList());
 	}
```

Comment:
In all other services the Message-ID contains the  '<' & '>'. I don't think we should treat this differently here.